cluster-scope/overlays/nerc-ocp-prod/kustomization.yaml

```diff
@@ -59,6 +59,7 @@ configMapGenerator:
   literals:
     - ack-4.11-kube-1.25-api-removals-in-4.12=true
     - ack-4.12-kube-1.26-api-removals-in-4.13=true
+    - ack-4.13-kube-1.27-api-removals-in-4.14=true
```

Comment:
These needs updating for a `ack-4.14...in-4.15` value, right?

Reply:
See https://docs.openshift.com/container-platform/4.15/updating/preparing_for_updates/updating-cluster-prepare.html#kube-api-removals_updating-cluster-prepare

> There are no Kubernetes API removals in OpenShift Container Platform 4.15.